pokemongo_bot/cell_workers/nickname_pokemon.py

```diff
@@ -0,0 +1,88 @@
+from pokemongo_bot import logger
+from pokemongo_bot.human_behaviour import sleep
+from pokemongo_bot.cell_workers.base_task import BaseTask
+
+class NicknamePokemon(BaseTask):
+    def initialize(self):
+        self.template = self.config.get('nickname_template','').lower().strip()
+        if self.template == "{name}":
+            self.template = ""
```

Comment:
This might seem unintuitive, but setting the nickname to blank (like in the app) will actually revert the pokemon back to its original name.  So, we set the template to blank to achieve this behavior when we anticipate that is what the user wants.